src/ir_Sharp.cpp

```diff
@@ -339,7 +340,20 @@ void IRSharpAc::on(void) { setPower(true); }
 void IRSharpAc::off(void) { setPower(false); }
 
 void IRSharpAc::setPower(const bool on) {
-  setBit(&remote[kSharpAcBytePower], kSharpAcBitPowerOffset, on);
+  if (on)
+    setBits(&remote[kSharpAcBytePower], kSharpAcBitPowerOffset, 2, kSharpAcOnFromOff);
```

Comment:
src/ir_Sharp.cpp:344:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
e.g.
```
    setBits(&remote[kSharpAcBytePower], kSharpAcBitPowerOffset,
            2, kSharpAcOnFromOff);
```
i.e. Break the line up (https://google.github.io/styleguide/cppguide.html#Line_Length) & line up the next (continued) line with what ever is logical. e.g. We split it inside the '()'s of `setBits()` so it wraps to the same column of the args to `setBits()` on the line above. i.e. With in it's scope as such. A general is, if there isn't a nice place to break the line, indent an _extra_ 4 spaces on the previous indent. Or, consider doing something else as your line is just too long or too indented! ;-)